### PR TITLE
[Modern UI] Login styles

### PR DESCRIPTION
**Pull request by cconard96**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

| Auror | Dark Auror | Midnight |
----------|----------------|------------|
| Screenshot from 2021-07-02 13-46-55 | Screenshot from 2021-07-02 13-46-34 | Screenshot from 2021-07-02 13-47-16 |

Changed:
- Fixed fetching of the palette. Not logged in pages were using the palette from `user_pref` which doesn't make sense if no user is logged in. For now, it just uses the default palette.
- Added ability to swap between light and dark logos in certain selectors

### CODE REVIEW

File: css/includes/_base.scss [hunk 1]
```diff
@@ -171,3 +177,12 @@ body.iframed {
...
 }
+
+.page-anonymous {
+   .glpi-logo {
+      --logo: url(#{$logo-dark});
+      content: var(--logo);
+      width: 200px;
+      height: 110px;
+   }
+}
```
Comment (orthagh) [inline css/includes/_base.scss:188]:
```suggestion
}

```

File: css/includes/_dark.scss [hunk 2]
```diff
@@ -156,3 +156,9 @@ input[type="color"] {
 body .modal-content, body .modal-header {
    background-color: $dark-mode-lighten;
 }
+
+.page-anonymous {
+   .glpi-logo {
+      --logo: url(#{$logo-light}) !important;
+   }
+}
```
Comment (orthagh) [inline css/includes/_dark.scss:164]:
```suggestion
}

```

**Review by orthagh — APPROVED** [review]

**Comment by nazaninhesari** [comment]
hi dear,how can you change logo ? i changed the logo in the folder pics but there is no change!

**Comment by trasher** [comment]
Please use [forums](http://forum.glpi-project.org) to get support.

**Pull request merged**